dali/kernels/signal/dct/dct_gpu.cu

```diff
@@ -60,8 +99,8 @@ __global__ void ApplyDct(const typename Dct1DGpu<OutputType, InputType>::SampleD
 template <typename OutputType, typename InputType>
 KernelRequirements Dct1DGpu<OutputType, InputType>::Setup(KernelContext &ctx,
                                                           const InListGPU<InputType> &in,
-                                                          span<const DctArgs> args,
-                                                          int axis) {
+                                                          span<const DctArgs> args, int axis,
+                                                          span<const float>) {
```

Comment:
We no longer pass unused arguments to setup. Please remove.

Reply:
done